packages/go/go-helpers.ts

```diff
@@ -32,46 +35,39 @@ const getGoUrl = (version: string, platform: string, arch: string) => {
   return `https://dl.google.com/go/go${version}.${goPlatform}-${goArch}.${ext}`;
 };
 
+const goGlobalCachePath = join(XDGAppPaths('com.vercel.cli').cache(), 'golang');
```

Comment:
Whats' the difference between this cache path and the on on line 22?

Reply:
The `cacheDir` is in the project directory and is used by `vc dev` to temporarily build and run the Go handlers.

`goGlobalCachePath` is where the Golang SDKs are installed and they are shared across projects as well as used to build the `analyze` binary on first use. The `goGlobalCachePath` is in the `com.vercel.cli` user data directory where package update metadata is stored.

Reply:
If the `cacheDir` is only used by `vc dev`, then it probably isn't need in `prepareCache()`. 

I think you need to add `goGlobalCachePath` to `prepareCache()`, right?